src/audio/kpb.c

```diff
@@ -91,6 +91,10 @@ struct comp_data {
 #ifdef CONFIG_IPC_MAJOR_4
 	struct ipc4_kpb_module_cfg ipc4_cfg;
 #endif /* CONFIG_IPC_MAJOR_4 */
+	uint32_t mic_sel_cnt_;
+	size_t num_of_in_channels_;
+	uint16_t offsets_[4];
+	struct kpb_micselector_config *mic_sel_;
```

Comment:
IMHO ending variable / structure names with an underscore is a part of the style and I don't think it's very popular in SOF in general and in this file in particular, I'd rather keep this uniform